modules/porous_flow/test/tests/poroperm/PermTensorFromVar01.i

```diff
@@ -0,0 +1,260 @@
+    type = PointValue
+    variable = perm_x
+    point = '2.5 0 0'
+  [../]
+  [./perm_y_right]
+    type = PointValue
+    variable = perm_y
+    point = '2.5 0 0'
+  [../]
+  [./perm_z_right]
+    type = PointValue
+    variable = perm_z
+    point = '2.5 0 0'
+  [../]
+[]
+
+[UserObjects]
+  [./dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pp'
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  [../]
+  [./pc]
+    type = PorousFlowCapillaryPressureVG
+    # unimportant in this fully-saturated test
+    m = 0.8
+    alpha = 1e-4
+  [../]
+[]
+
+[Modules]
+  [./FluidProperties]
+    [./simple_fluid]
+      type = SimpleFluidProperties
+    [../]
+  [../]
+[]
+
+[Materials]
+  # Permeability
+  [./permeability]
+    type = PorousFlowPermeabilityTensorFromVar
+    k_anisotropy = '1 0 0  0 2 0  0 0 0.1'
+    perm = perm_var
+  [../]
+  [./temperature]
+    type = PorousFlowTemperature
+  [../]
+  [./temperature_nodal]
+    type = PorousFlowTemperature
+    at_nodes = true
+  [../]
+  [./massfrac]
+    type = PorousFlowMassFraction
+    at_nodes = true
+  [../]
+
+  # Fluid pressure
+  [./eff_fluid_pressure]
+    type = PorousFlowEffectiveFluidPressure
+  [../]
+  [./ppss]
+    type = PorousFlow1PhaseP
+    porepressure = pp
+    capillary_pressure = pc
+  [../]
+  [./ppss_nodal]
+    type = PorousFlow1PhaseP
+    at_nodes = true
+    porepressure = pp
+    capillary_pressure = pc
+  [../]
+
+  # Fluid properties
+  [./simple_fluid]
+    type = PorousFlowSingleComponentFluid
+    fp = simple_fluid
+    phase = 0
+    at_nodes = true
+  [../]
+  [./simple_fluid_qp]
+    type = PorousFlowSingleComponentFluid
+    fp = simple_fluid
+    phase = 0
+  [../]
+  [./dens_all]
+    type = PorousFlowJoiner
+    at_nodes = true
+    material_property = PorousFlow_fluid_phase_density_nodal
+  [../]
+  [./dens_all_at_quadpoints]
+    type = PorousFlowJoiner
+    material_property = PorousFlow_fluid_phase_density_qp
+  [../]
+  [./visc_all]
+    type = PorousFlowJoiner
+    at_nodes = true
+    material_property = PorousFlow_viscosity_nodal
+  [../]
+
+  # Porosity
+  [./porosity]
+    type = PorousFlowPorosity
+    porosity_zero = 0.1
+  [../]
+
+  # Permeability
+  [./relperm]
+    type = PorousFlowRelativePermeabilityCorey
+    at_nodes = true
+    n = 0 # unimportant in this fully-saturated situation
+    phase = 0
+  [../]
+  [./relperm_all]
+    type = PorousFlowJoiner
+    at_nodes = true
+    material_property = PorousFlow_relative_permeability_nodal
+  [../]
+[]
+
+[Preconditioning]
+  [./andy]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  solve_type = Newton
+  type = Steady
+  l_tol = 1E-5
+  nl_abs_tol = 1E-3
+  nl_rel_tol = 1E-8
+  l_max_its = 200
+  nl_max_its = 400
+
+  petsc_options_iname = '-pc_type -pc_asm_overlap -sub_pc_type -ksp_type -ksp_gmres_restart'
+  petsc_options_value = ' asm      2              lu            gmres     200'
+[]
+
+[Outputs]
+  file_base = PermTensorFromVar01
```

Comment:
Unnecessary parameter. Applies to most (or all) of these new test cases.

Reply:
I'm confused. Surely its needs a file name in order to perform CSVDiff?